### PR TITLE
Ciel Day 3+4 Edits

### PR DESCRIPTION
**Pull request by Parzivus**
This starts at the choice after killing the white woman, so 3_3_2 is the first file. Not too much is changed, as these files are already pretty well edited. I'll try to get at least day 5 done fairly soon as well.

### CODE REVIEW

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt [hunk 1]
```diff
@@ -350,7 +351,8 @@ I was lying to myself when I said I couldn't think straight.
 [sha:0d72519dbf38b12672c3c6cdd9ab88ecda651630]{
 -- Page 149, Offset 5230.
 -- だってさっきからそうしたくて、俺は何もしなかった。
-I've wanted to this whole time, I just haven't.
+// Is this closer to the intended meaning that the comment in the previous line mentioned? Need a JP check.
+I could think this whole time, I just haven't.
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:355]: Original is correct, sentence conveys intent not ability. (though this change makes more sense from an english perspective, agreed).

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt [hunk 2]
```diff
@@ -151,6 +151,7 @@ In my heart I still fear death,
 [sha:aefd2d0b8f749a4654f00ffc113ad7a52f422f1e]{
 -- Page 135, Offset 5192.
 -- ざあざあ、という雑音。
+// "rhythm" or "patter" would work just as well and aren't so obscure (and sus)
 A soft, steady susurration.
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:155]: Patter works for me, but the alliteration is too good. I'm for keeping susurration.

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:155]: alliteration is all well and good but "susurration" is a a word so terrible, the firefox spellchecker thinks its a typo. I'm for "patter"

File: script/Ciel/Day 3/03_01_CIEL03_3_3B.txt [hunk 3]
```diff
@@ -92,7 +92,8 @@ When I try to shake her off―――
 [sha:67d9cae6e82c27cd9eb8e0a3ed4a7ced04800a87]{
 -- Page 198, Offset 5355.
 -- 何の悪意もない偶然で、世界の在り方が一変した。
-Through no fault of anyone's, the world is turned on its head.
+// Requesting check on whether the original or the edit is more commonly used
+Through no fault of anyone, the world is turned on its head.
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_3_3B.txt:96]: new line is correct

File: script/Ciel/Day 3/03_01_CIEL03_3_3B.txt [hunk 4]
```diff
@@ -226,8 +227,8 @@ I turned that place into hell just by being there.
 [sha:c112bad1052087501fcd15b3b89cd9eea9d0696b]{
 -- Page 204, Offset 5381.
 -- だから俺が、理由もなく何かを台無しにしてしまうのはごく自然な在り方なのだと、心の何処かで―――
-That's why, somewhere in my heart,
- I know that it's in my nature to wantonly destroy everything around me―――
+That's why I know, somewhere in my heart,
+ that it's in my nature to wantonly destroy everything around me―――
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_3_3B.txt:231]: prefer old, has a more poetic sense to it

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:155]: Obviously partial to susurration, but if it's too SAT word we can tone it down

File: script/Ciel/Day 4/04_01_CIEL04_1B.txt [hunk 5]
```diff
@@ -390,7 +391,8 @@ Please take care on your way to school today."
 [sha:61e7ddc7a7c36ec8042a71855874e5d64e739e1a]{
 -- Page 68, Offset 8334.
 -- 翡翠は深々と礼をして俺を送り出してくれた。
-Bowing especially deeply, she sends me on my way.
+I thank Senpai from the bottom of my heart for that smile of hers,
+ and begin to make my way back to the Tohno mansion.
```
Comment (rschlaikjer) [inline script/Ciel/Day 4/04_01_CIEL04_1B.txt:395]: Uh what's goin on here? this is a totally different line

**Review by rschlaikjer — APPROVED** [review]

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt (hunk 2 above)
Comment (Parzivus) [reply script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:155]: I'll leave it for the group PR

File: script/Ciel/Day 4/04_01_CIEL04_1B.txt (hunk 5 above)
Comment (Parzivus) [reply script/Ciel/Day 4/04_01_CIEL04_1B.txt:395]: Not sure what happened here, I'll just revert it

**Pull request merged**